Mcu/l431/Src/peripherals.c

```diff
@@ -71,17 +71,15 @@ void SystemClock_Config(void)
 #endif
 
 #else
-  LL_RCC_MSI_Enable();
+  LL_RCC_HSI_Enable();
 
    /* Wait till MSI is ready */
-  while(LL_RCC_MSI_IsReady() != 1)
+  while(LL_RCC_HSI_IsReady() != 1)
   {
 
   }
-  LL_RCC_MSI_EnableRangeSelection();
-  LL_RCC_MSI_SetRange(LL_RCC_MSIRANGE_6);
-  LL_RCC_MSI_SetCalibTrimming(0);
-  LL_RCC_PLL_ConfigDomain_SYS(LL_RCC_PLLSOURCE_MSI, LL_RCC_PLLM_DIV_1, 40, LL_RCC_PLLR_DIV_2);
+  LL_RCC_HSI_SetCalibTrimming(16);
```

Comment:
why 16 for the trimming? maybe this is specific to your board?

Reply:
If I remember correctly that value comes from ST's data sheet and is mentioned there as default value. I did not do any trimming specific to my hardware.